variables.mk

```diff
@@ -7,3 +7,6 @@ BUSYBOX_VERSION ?= 1.31.1
 # This is a workaround for syntax highlighters that break on a "Comment" symbol.
 HASHSIGN = \#
 SLASHSIGN = /
+
+# for unit tests
+BATS_BIN_PATH = tests/test/libs/bats/bin/bats
```

Comment:
In my opinion WHATEVER_PATH variable should contain path-like contents, ie path to a directory.